CHANGELOG.md

```diff
@@ -1,74 +1,83 @@
-# Change Log for Tensile
+# Changelog for Tensile
 
 ## (Unreleased) Tensile 4.42.0
 
-### Additions
-- add contributor and developer guide
-- add testing and documentation for MasterSolutionLibrary.ArchitectureIndexMap and remapSolutionIndicesStartingFrom
-- add gfx12 support
-- add functions for writing master file
-- add tPrint and reconciles printing options
-- add Python unit test coverage report
-- add factor embed library logic into function and test
-- add clang++ as cxx-compiler option for windows
-- add logic to cope with different compilers
-- add generateManifest fxn and rename generateManifest to toFile and move to Utilities
-- add profiling CI job
-- add support for amdclang and use defaults
-- add architecture management functions to TensileCreateLibrary
-- add TensileCreateLibrary cli reference docs
-- add new documentation (sphinx prototype, build out skeleton)
+### Added
 
-### Optimizations
-- add prediction model for optimal number of Stream-K tiles to run
-- use analytical grid size prediction model for Stream-K
-- remap XCC-based workgroup for Stream-K kernels
-- add two-tile algorithm with Stream-K after DP
-- add atomic 2-tile Stream-K and clean-up tuning parameters
+- Contributor and developer guide
+- Testing and documentation for `MasterSolutionLibrary.ArchitectureIndexMap` and `remapSolutionIndicesStartingFrom`
+- gfx12 support
+- Functions for writing master file
+- `tPrint` and reconcile printing options
+- Python unit test coverage report
+- Factor embed library logic into function and test
+- `clang++` as cxx-compiler option for Windows
+- Logic to cope with different compilers
+- `generateManifest` function and rename generateManifest to toFile and move to Utilities
```

Comment:
@bstefanuk If we have renamed generateManifest function to `toFile` already then why are we mentioning that we have added generateManifest function? Shouldn't we write "Added toFile function in Utilities".

Reply:
This update is fine, but please mention that the generate manifest step has been moved into this function as well.

Reply:
done.